doc/api.md

```diff
@@ -175,7 +175,7 @@ HTTP/1.1 200 OK
 |`polling_interval`         |An interval, in milliseconds, at which Fireworq checks the arrival of new jobs in this queue.|optional, defaults to [`FIREWORQ_QUEUE_DEFAULT_POLLING_INTERVAL`][env-queue-default-polling-interval]|
 |`max_workers`              |The maximum number of jobs that are processed simultaneously for this queue.|optional, defaults to [`FIREWORQ_QUEUE_DEFAULT_MAX_WORKERS`][env-queue-default-max-workers]|
 |`max_dispatches_per_second`|The maximum floating-point number of dispatches allowed to be processed within a second for this queue.|optional, defaults to no throttling. When throttling is configured, `polling_interval` is fixed to `100` regardless of the default interval|
-|`max_burst_size`           |The maximum number of burst size of throttling configuration for this queue.|optional, cannot be set without `max_dispatches_per_second`|
+|`max_burst_size`           |The maximum number of burst size of throttling configuration for this queue.|Must be set with `max_dispatches_per_second`, cannot be set without `max_dispatches_per_second`|
```

Comment:
Both with and without looks redundant to me, so how about simply
```suggestion
|`max_burst_size`           |The maximum number of burst size of throttling configuration for this queue.|optional, configured with `max_dispatches_per_second`|
```

Reply:
Nice phrase! I adopted it.